docs/introduction/sla.md

```diff
@@ -0,0 +1,208 @@
+
+ * Issues regarding CSC’s identity management service
+ * Issues using this service’s web interface
+ * Issues using the provided APIs
+ * Issues deploying workloads
+ * Issues using workload templates
+
+Response and fulfilment times are provided as service level targets (see section 6).
+
+## Service level targets
+
+The Rahti service level targets adhere to JHS1745 as follows:
+
+Service level targets ||
+--- | ---
+Service Level | B (normal)
+Service time / incident handling | P2 Weekdays 7.00-19.00
+Availability | K2 99%
+Response | V2 reaction: 2h, solution: 1 BD
+
+Availability is calculated by subtracting the service break time from the ideal
+availability during service time. This information is obtained from internal
+monitoring systems and defined as Customers’ ability to manage their workloads,
+and the availability of the workloads.
+
+The Service Provider commits to inform the customer if this SLA is violated or
+a violation is anticipated. For this, email as a communication channel will be
+used. 
+
+A Customer contacts the CSC Service Desk for the case of a possible SLA
+violation. The case will be analysed internally and, if the violation is
+confirmed, CSC will inform the Customer about the reasons for the violation,
+planned mitigation actions and expected resolution time.
+
+## Limitations and constraints
+
+The provisioning of the service under the agreed service level targets is subject to the following limitations and constraints: 
+
+ * Support is provided primarily in the following languages: English, Finnish
+ * Downtimes caused due to upgrades for fixing critical security issues are not considered SLA violations. In the case of critical security upgrades CSC reserves the right to apply the upgrades with minimal notice
+ * A technical failure which affects individual customers does not count as downtime
+ * Failures of single instances of the workload (containers) are expected behaviour, and does not count as downtime
+ * Any other causes outside service provider’s direct control
+
+## Communication, reporting and escalation
+
+### General communication
+
+The following contacts will be generally used for communications related to the
+service in the scope of this SLA:
+
+| Communication channels ||
+--- | ---
+Contact for Customers | servicedesk@csc.fi
+Security incident reporting | (CSC’s Head of Security) security@csc.fi +358 (0) 94 57 2253
+
+### Reporting
+The service availability information is…. The information provided will be limited to service availability and by data security and privacy constraints.
+
+### Escalation and complaints
+
+For escalation and complaints, the defined contact point shall be used, and the
+following rules apply:
+
+1. First contact shall be established, preferably by e-Mail, to Contact for Customers address (See section 8.1) explaining the reason for the complaint with a sensible level of detail and clarity. Please also include, if possible, the following information:
```

Comment:
There's a reference to section 8.1, but that doesn't exist in the markdown document.

Reply:
Fixed all the references to markdown references